internal/pktline/capabilities_test.go

```diff
@@ -73,3 +73,26 @@ func TestParseCapabilitiesWithArguments(t *testing.T) {
 		)
 	}
 }
+
+func TestSafeCapabilityValue(t *testing.T) {
```

Comment:
What do you think about writing the test like this?

```suggestion
func TestSafeCapabilityValue(t *testing.T) {
	for _, p := range []struct {
		val      string
		expected bool
	}{
		{"", true},
		{"AA:BB:CC:01", true},
		{"abcdefg", true},
		{"not valid", false},
		{"not\tvalid", false},
		{"not\rvalid", false},
		{"not\nvalid", false},
	} {
		t.Run(
			fmt.Sprintf("TestSafeCapabilityValue(%s)", p.val),
			func(t *testing.T) {
				assert.Equal(t, p.expected, IsSafeCapabilityValue(p.val))
			},
		)
	}
}
```

Reply:
Yeah, I can do that, thanks for the suggestion! I accepted the change and then modified it in 1ef340530c57bba7a3465f621f3396e9576780fe... I like defining the examples before the `for`, I think it's easier to follow. I also dropped the t.Run, I try to avoid using t.Run if its body is very simple like here, I find it easier to read the test and the output if it's all in the top-level test case.